drive/driveimpl/fileserver.go

```diff
@@ -41,18 +46,36 @@ func NewFileServer() (*FileServer, error) {
 	// TODO(oxtoacart): actually get safesocket working in more environments (MacOS Sandboxed, Windows, ???)
 	l, err := net.Listen("tcp", "127.0.0.1:0")
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("listen: %w", err)
 	}
 	// }
```

Comment:
I'd remove this stray `// }`, or put it up above with the rest of the commented-out code.